console/src/components/Services/Data/Add/Tooltips.js

```diff
@@ -33,3 +33,10 @@ export const uniqueKeyDescription = (
 export const dataTypeDescription = description => (
   <Tooltip id="tooltip-datatype-description">{description}</Tooltip>
 );
+
+export const checkConstraintsDescription = (
```

Comment:
Same. This tooltip should be reused between `Add` and `Modify`.

Reply:
I did a small refactor, because in `Modify` we are using component [`ToolTip`](https://github.com/hasura/graphql-engine/blob/master/console/src/components/Common/Tooltip/Tooltip.js), which is more convenient, so I replaced bootstrap `OverlayTrigger` and `Tooltip` with our `ToolTip` component on `Add`.